sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 4.4.0-beta.8 (Unreleased)
```

Comment:
Should this be 4.4.0 or are you planning to remove the beta postfix in a follow-up commit?

Reply:
I will update the CHANGELOG in a separate PR.